assets/css/less/wpbdp.less

```diff
@@ -152,7 +152,7 @@ form#wpbdmsearchform a.advanced-search-link {
 
 .wpbdp-listing-excerpt .listing-thumbnail {
     float: right;
-    margin: 0 0 0 10px;
+    margin: 0 auto;
```

Comment:
It looks like this centers the image one smaller screens when it should be aligned left. I think margin:0 looks like it would work better.